docs/01-overview/api-exposure/README.md

```diff
@@ -53,7 +52,6 @@ If you're interested in learning more about API Exposure in Kyma, follow these l
 - Troubleshoot API Exposure-related issues when:
 
   - You [cannot connect to a service exposed by an APIRule](../../04-operation-guides/troubleshooting/api-exposure/apix-01-cannot-connect-to-service/apix-01-01-apigateway-connect-api-rule.md)
-  - You get the [`401 Unauthorized` or `403 Forbidden`](../../04-operation-guides/troubleshooting/api-exposure/apix-01-cannot-connect-to-service/apix-01-02-401-unauthorized-403-forbidden.md) status code when you try to connect to a service exposed by an APIRule
```

Comment:
i think this one should not be removed. instead troubleshooting guide should be adjusted and more general